base/views.jl

```diff
@@ -132,6 +132,7 @@ end
 @propagate_inbounds maybeview(A, args...) = getindex(A, args...)
 @propagate_inbounds maybeview(A::AbstractArray, args...) = view(A, args...)
 @propagate_inbounds maybeview(A::AbstractArray, args::Number...) = getindex(A, args...)
```

Comment:
```suggestion
@propagate_inbounds maybeview(A::AbstractArray, args::Union{Number, CartesianIndex}...) = getindex(A, args...)
```

Reply:
thanks, I didn't even know mixing indices was allowed in julia